### ISSUE TITLE
Argument list too long

### ISSUE THREAD

**Issue by besarthoxhaj** [redbooth/react-native-auto-updater#11]
Xcode keeps throwing the following error: `Argument list too long: recursive header expansion failed at /Users/besartshyti/Desktop/Awesome/node_modules/react-native-auto-updater/ios/../../react-native/node_modules/babel-plugin-transform-es2015-computed-properties/node_modules/babel-template/node_modules/babel-traverse/node_modules/babel-code-frame/lib.`

Just started a new project and installed through npm. Tried to link the libraries with `rnpm link` and also manually following the README. Nothing so far.

Any suggestion?

I'm using react-native 0.19

**Comment by rahuljiresal** [comment]
@besarthoxhaj I haven't upgraded to React Native 0.19 yet, but remove the "recursive" option in the "Search Header Paths" for the project and see if it works?

I suspect that this problem is arising because the way react-native 0.19 and npm 3.x organizes packages. I have a feeling it is creating a symlink to a parent folder at some level.

**Comment by izaakrogan** [comment]
@besarthoxhaj @rahuljiresal I got the same problem. Will try to downgrade react-native and see what happens.

However I would like to use 0.19. Is there some way to fix this? I tried remove "recursive" but nothing.

**Comment by besarthoxhaj** [comment]
@rahuljiresal got the same error on 0.18. Am I doing something else wrong?

**Comment by rahuljiresal** [comment]
@besarthoxhaj Upgraded to RN 0.19 and using npm v 3.3.12. Works fine on the old Example project I have set up on my machine. Also tried a new project from scratch. Builds and runs perfectly fine.

Did you try Cleaning the project, quitting Xcode and restarting it?

**Comment by besarthoxhaj** [comment]
@rahuljiresal tried again today, no luck! I will keep going and hopefully come and report the solution.

**Comment by xuming314** [comment]
in your project, select RectNativeAutoUpdater.xcodeproj  then select  Build Settings->Search Paths->Header Search Paths->keep "$(SRCROOT)/../../react-native/React   recursive" only. delete another path. 
it's will ok.

**Comment by rjbaker** [comment]
+1 Also got this error with React Native 0.19.0

```
Argument list too long: recursive header expansion failed at /Users/rjbaker/Documents/Development/ios/HazardsReact/node_modules/react-native-auto-updater/ios/../../react-native/node_modules/babel-plugin-transform-es2015-computed-properties/node_modules/babel-helper-define-map/node_modules/babel-helper-function-name/node_modules/babel-helper-get-function-arity.
```

**Comment by rjbaker** [comment]
Fixed this in pull request #13

**Comment by besarthoxhaj** [comment]
thanks @xuming314 that solved my problem!
- create a new project `awesome`
- installed `react-native-auto-updater`
- followed the instruction to link the library
- failed with error `Argument list too long`
- went to `RectNativeAutoUpdater.xcodeproj`
- deleted all the search paths and added `$(SRCROOT)/../../react-native/React` _recursive_

it worked!

**Comment by besarthoxhaj** [comment]
@rahuljiresal I saw your comment on https://github.com/aerofs/react-native-auto-updater/pull/13 

I'm a bit confused, is this not the right solution?

**Comment by rahuljiresal** [comment]
This issue has been fixed and a new version 0.2.1 has been pushed to npm.
